configuration/rules-ng.md

```diff
@@ -374,24 +376,26 @@ There are several ways to add new rules:
 
 `automation exportRules [-p] <parserType> [-st] <file> ` - Exports Rules in a file. If parser type missing, "json" parser will be set as default.
 
-`automation enableRule [-st] <uid> <enable> ` - Enables the Rule, specified by given UID. If enable parameter is missing, the result of the command will be visualization of enabled/disabled state of the rule, if its value is "true" or "false", the result of the command will be to set enable/disable on the Rule.
+`automation enableRule [-st] <uid> <enable> ` - Enables the Rule, specified by given UID.
```

Comment:
Enables the Rule, specified by the given UID. The use of the 'enable' argument is optional, and will accept a boolean value. If used, the command will enable (true) or disable (false) the Rule. If it is not used, the command will return the current status of the Rule.